### PR TITLE
The code of conduct

### PR DESCRIPTION
**Pull request by aliveevie**
PR Checklist:

- [ ] make all edits to the docs in the `src` directory, not in `docs` or in the html files
- [ ] note any issues closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
- [ ] put any other relevant information below

### CODE REVIEW

File: aliveevie/code_of_conduct.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# code of conduct and reporting guidelines
+# Teams & Roles
```
Comment (kathatherine) [inline aliveevie/code_of_conduct.md:2]: And another blank line is needed beneath this header as well.

File: aliveevie/code_of_conduct.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+- **maintainers:** A maintainer is an individual responsible for the management of one or more feedstock repositories and packages. Maintainers have the ability to merge pull requests into the feedstock of only the packages they maintain.
+
+- **external contributors:** This group encompasses all others who are not on core, part of staged-recipes, or maintainers. This includes first-time contributors, collaborators, and funders. They have no special rights within the conda-forge organization itself.
+
+- **emeritus-core:** Core members that are inactive (commits, GitHub comments/issues/reviews, dev meetings and voting on polls) in the past six months will be asked if they want to become emeritus-core developers. Any core member can also request to become emeritus if they wish to do so (e.g. taking a sabbatical or long vacation). Emeritus core members can still vote and move back to active core anytime. Emeritus votes are used to count towards quorum but the quorum size is computed from the size of the active core group. The core.csv list should be updated when a change in the status of a member occurs.
+
```
Comment (kathatherine) [inline aliveevie/code_of_conduct.md:13]: Did you get the information above from anywhere in particular or did you do some research and put it together?

File: aliveevie/code_of_conduct.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Sub-Teams
```
Comment (kathatherine) [inline aliveevie/code_of_conduct.md:14]: You also need a blank line beneath this header.

**Review by kathatherine — CHANGES_REQUESTED** [review]
Thanks so much! You should also make this a .rst file instead of a .md file.

**Comment by jaimergp** [comment]
https://github.com/conda-forge/conda-forge.github.io/pull/1640

**Pull request closed without merging**